SimPEG/electromagnetics/frequency_domain/fields.py

```diff
@@ -320,7 +320,7 @@ def _jDeriv(self, src, du_dm_v, v, adjoint=False):
 
 class Fields3D_e(FieldsFDEM):
```

Comment:
Do you think it is worth updating the names of the fields objects here to match that of the simulation? 
```
Fields3DElectricField
```